camsys/transam_lib
# Transam dependencies

* Make travis get latest versions of depended-on engines
* Make sure Gemfile.lock is not committed
* Add mysql2 to Gemfile to fix testing
